packages/realm/src/Object.ts

```diff
@@ -274,13 +274,15 @@ export class RealmObject<T = DefaultObject> {
    */
   private declare readonly [KEY_SET]: ReadonlySet<string>;
 
+  /** @deprecated Please use {@link https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/keys|Object.keys()}  */
```

Comment:
Spaces needed in between `|` for the formatting to be correct 👍 
```suggestion
  /** @deprecated Please use {@link https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/keys | Object.keys()}  */
```

Reply:
Thanks!